app/Http/Controllers/PhotoController.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\repositories\PhotoAlbumRepository;
+use App\repositories\RepositorieFactory;
+
+class PhotoController extends Controller
+{
+    private $_PhotoAlbumRepository;
+    private $_PhotoRepository;
+
+    public function __construct(RepositorieFactory $repositorieFactory){
+        $this->_PhotoAlbumRepository = $repositorieFactory->getRepositorie(RepositorieFactory::$PHOTOALBUMREPOKEY);
+        $this->_PhotoRepository = $repositorieFactory->getRepositorie(RepositorieFactory::$PHOTOREPOKEY);
+        $this->middleware('auth');
+    }
+
+    public function index($id)
+    {
+        PhotoAlbum:$photoAlbum = $this->_PhotoAlbumRepository->find($id);
+        $photoAlbum->description = str_replace("\r\n","<br>", $photoAlbum->description);
+        $thumbnails = $this->getThumbnails($id);
+        $photos = $this->getPhotos($id); 
+        $curPageName = $photoAlbum->title . " - Eindhovense Studenten Alpen Club";
```

Comment:
Nu staat 'eidhovense studenten Alpen club' dubbel in de titel, alleen de album naam is genoeg.

Reply:
fixed in commit 7ea5cfa